src/Navigation/Back.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * @Author: Erik Slovak <erik.slovak@technologystudio.sk>
+ * @Date: 2022-11-07T12:11:78+01:00
+ * @Copyright: Technology Studio
+**/
+
+import { Log } from '@txo/log'
+
+import type {
+  OnActionAttributes,
+} from '../Model/Types'
+
+const log = new Log('txo.react-conditional-navigation.Navigation.Back')
+
+export const onBackAction = ({
+  action,
+  originalOnAction,
+  restArgs,
+}: OnActionAttributes): boolean => {
+  // TODO: add backToRouteName, key and routeName when necessary
```

Comment:
Is this forgotten or ?

Reply:
It's not forgotten, but it's unnecessary, removed.